BitbucketServerAutomation/Functions/Get-BBServerChanges.ps1

```diff
@@ -56,22 +56,18 @@ function Get-BBServerChanges
     $lastPage = $false
     $nextPageStart = 0
     $changes = $null
-    $changesList = @()
 
     while ( -not $lastPage )
     {
-        $resourcePath = ('projects/{0}/repos/{1}/compare/changes?from={2}&to={3}' -f $ProjectKey, $RepoName, $From, $To)
+        $resourcePath = ('projects/{0}/repos/{1}/compare/changes?from={2}&to={3}' -f $ProjectKey, $RepoName, [Web.HttpUtility]::UrlEncode($From), [Web.HttpUtility]::UrlEncode($To))

```

Comment:
Will you add a test for this? Create from and to branches that have a character that needs to be encoded, e.g. `feature/one`, `release/major`, etc.